demisto_sdk/commands/init/templates/BaseIntegration/BaseIntegration.yml

```diff
@@ -0,0 +1,52 @@
+category: Utilities
+commonfields:
+  id: BaseIntegration
+  version: -1
+configuration:
+- defaultvalue: https://example.com/
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  additionalinfo: The API Key to use for connection
+  name: apikey
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
```

Comment:
I don't like the fact that we copy the files from content repo, because then we will not get updates on these templates.
Maybe as part of the demisto-sdk version release, these templates will be downloaded and added to the project. This way each release demitso-sdk will contain the updated version of the templates.

Reply:
These files are only backups in case the files extraction from content fails

Reply:
@anara123 - that is a good idea though to update the templates we should open an issue about it especially since the SDK release automation is in progress - https://github.com/demisto/etc/issues/22907
For now keeping the integration on hand as a fallback does seem like a good idea to me.